CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
My plan is to release this as `2.0.0`. Tag the previous commit as `1.0.0` and then with Go modules we can use that simply as `v2`. And with dep we can specify `=1.0.0` or `>=2.0.0` so there is no need to do all the migrations at once.

Reply:
Small correction here. The versions here should be 0.1.0 and 0.2.0.